iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ShuntCompensator.java

```diff
@@ -17,71 +17,79 @@
 public interface ShuntCompensator extends Injection<ShuntCompensator> {
 
     /**
-     * Get the maximum section count.
+     * Get the current count of sections in service.
+     * Please note sections can only be sequentially in service i.e. all sections from section 1 to section currentSectionCount are in service.
+     * <p>
+     * It is expected to be greater than one and lesser than or equal to the
+     * maximum section count.
+     * <p>
+     * Depends on the working variant.
+     * @see VariantManager
+     */
+    int getCurrentSectionCount();
```

Comment:
Maybe we could try to find a better naming than current. What I don't like with current is that in power system it is also a unit.

Reply:
Maybe `getActivatedSectionCount()`?

Reply:
Should we also rename `getCurrentB()` and `getCurrentG()`? What do you propose for those?